ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/AvailableVariable.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.shell.invoker;
+
+/**
+ * Representation for available variables.
+ * Holds information about variable name, type and
+ * current value.
+ *
+ * @since 2.0.0
```

Comment:
Shall we update this to correct version ? Check other files as well.

Reply:
Updated in [e42491f](https://github.com/ballerina-platform/ballerina-lang/pull/35871/commits/e42491fff2d9ae4b0a7864fe3a1cdd22af6e0dcd).